src/org/ods/util/HelmStatusSimpleData.groovy

```diff
@@ -0,0 +1,186 @@
+package org.ods.util
+
+import com.cloudbees.groovy.cps.NonCPS
+import groovy.transform.TypeChecked
+
+@TypeChecked
+class HelmStatusSimpleData {
```

Comment:
This is just a typical JavaBean holding a set of properties. It's reasonable to have a builder from a Map, but otherwise, no other logic or validation should be done. The users of this data will validate the property values when needed.

Reply:
I'd like to ask for some clarification on this, regarding use cases for this HelmStatusSimpleData.
If this class is intended to be used as a "typed" JavaBean object in order to guarantee type-safety and data integrity, excluding the logic for converting to/from Map, what's the purpose for the rest of the logic?

Reply:
Hi @anteloro-boeh I am back now and if needed could answer questions about what I had in mind when writing this code in the first place. But perhaps this was already sorted out.?